client/src/components/style.css.js

```diff
@@ -0,0 +1,57 @@
+export default {
+  container: {
+    display: 'flex',
+    position: 'absolute',
+    'justify-content': 'center',
```

Comment:
you don't have to change this, but you could also leave the quotes off the keys by camel casing i.e. 'justify-content' = justifyContent

Reply:
I agree, and React will give you a warning here, you should use `justifyContent`